platformio/lib/OneWire/OneWire.cpp

```diff
@@ -0,0 +1,62 @@
+#include "OneWire.h"
+
+void OneWire::reset()
+{
+    _pin.output();
+    _pin = 0;
+    wait_us(500);
+    _pin.input();
+    wait_us(500);
+}
+
+void OneWire::write_bit(bool bit)
+{
+    if (bit) {
+        _pin.output();
+        _pin = 0;
+        wait_us(5);
+        _pin.input();
+        wait_us(55);
+    } else {
+        _pin.output();
+        _pin = 0;
+        wait_us(60);
+        _pin.input();
+    }
+}
+
+// output byte d (least sig bit first).
+void OneWire::write_byte(unsigned char d)
+{
+    for (uint8_t n=8; n!=0; n--)
```

Comment:
Hmm... I think I had a different approach in at some place, where I used the `n`. But it doesn't make any sense this way.